Pokedex/Pokedex/View Controllers/PokemonDetailViewController.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  PokemonDetailViewController.swift
+//  Pokedex
+//
+//  Created by Cameron Dunn on 1/25/19.
+//  Copyright © 2019 Cameron Dunn. All rights reserved.
+//
+
+import UIKit
+
+class PokemonDetailViewController: UIViewController, UISearchBarDelegate, UICollectionViewDelegate, UICollectionViewDataSource {
+    
+    
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var idLabel: UILabel!
+    @IBOutlet weak var typeLabel: UILabel!
+    @IBOutlet weak var abilitiesLabel: UILabel!
+    @IBOutlet weak var button: UIButton!
+    
+    var model : Model?
+    var pokemon : Pokemon?
+    
+    
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        var imageCount = 0
+        
+        return 1
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "Cell", for: indexPath) as! CustomCollectionViewCell
+        return cell
```

Comment:
You need to pass the pokemon to the custom cell class and update it's image
```suggestion
        guard let poke = pokemon else { return cell }
        cell.pokemon = poke
        return cell
```